pkg/config/lima_config_applier.go

```diff
@@ -154,7 +154,18 @@ func toggleUserModeEmulationInstallationScript(limaCfg *limayaml.LimaYAML, enabl
 			Mode: "system",
 			Script: fmt.Sprintf(`%s
 #!/bin/bash
-dnf install -y --setopt=install_weak_deps=False qemu-user-static-aarch64 qemu-user-static-arm qemu-user-static-x86
+qemu_pkgs=""
+if [ ! -f /usr/bin/qemu-aarch64-static ]; then
+  qemu_pkgs="$qemu_pkgs qemu-user-static-aarch64"
+elif [ ! -f /usr/bin/qemu-aarch64-static ]; then
+  qemu_pkgs="$qemu_pkgs qemu-user-static-arm"
+elif [ ! -f  /usr/bin/qemu-aarch64-static ]; then
+  qemu_pkgs="$qemu_pkgs qemu-user-static-x86"
+fi
+
+if [[ $qemu_pkgs ]]; then
+  dnf install -y --setopt=install_weak_deps=False ${qemu_pkgs}
```

Comment:
nit: I think the current solution should be 100% fine for this current case, but I think it'd be a little safer to have `qemu_pkgs` be an array, e.g.

```
qemu_pkgs=()
qemu_pkgs=("${qemu_pkgs[@]}" "qemu-user-static-aarch64")
dnf install -y --setopt=install_weak_deps=False "${qemu_pkgs[@]}"
```

just because it allows us to quote the variable expansion.